Dockerfile

```diff
@@ -9,7 +9,8 @@ RUN apt-get update \
 
 ENV LIFERAY_HOME=/usr/local/liferay-ce-portal-7.0-ga5
 ENV LIFERAY_SHARED=/storage/liferay
-ENV LIFERAY_WEB_SERVER_PROTOCOL=https
+ENV LIFERAY_WEB_PERIOD_SERVER_PERIOD_PROTOCOL=https
```

Comment:
I've been thinking it over and it does make sense to apply these at runtime.  Since the default is http it would work fine locally out of the box and when we publish them to wedeploy we can just add it as an environment variable now.